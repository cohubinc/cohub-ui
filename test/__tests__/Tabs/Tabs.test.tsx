import React from "react";
import { shallow } from "enzyme";

import { Tabs } from "dist";

it("renders without crashing", async () => {
  const wrapper = shallow(
    <Tabs useRedux={false}>
      <Tabs.Tab title="test" />
      <Tabs.Tab title="Test 2" />
    </Tabs>
  );

  expect(wrapper).toMatchSnapshot();
});
