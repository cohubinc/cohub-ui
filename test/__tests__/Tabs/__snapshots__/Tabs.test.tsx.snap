// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
<Fragment>
  <div
    className="Tabs flex"
  >
    <Tab
      key=".0"
      title="test"
      useRedux={false}
    />
    <Tab
      key=".1"
      title="Test 2"
      useRedux={false}
    />
  </div>
  <div
    className="Tabs-Content"
  />
</Fragment>
`;
